pages/PriceComparision.tsx

```diff
@@ -0,0 +1,18 @@
+import { KakaoMap } from "@shared/components/map";
+import { useState } from "react";
+import styled from "styled-components";
+
+
+export const PriceComparision: React.FC = () => {
```

Comment:
`page routing`을 사용하려면 default export를 사용해야 합니다!
```suggestion
const PriceComparision = () => {

// 나머지 코드

}
export default PriceComparision
```

Reply:
감사합니다!